Cargo.toml

```diff
@@ -34,7 +36,7 @@ prost = { version = "0.12" }
 prost-types = { version = "0.12" }
 serde_json = { version = "1.0.111" }
 uuid = { version = "1.7.0" }
-up-rust = { default-features = false, git = "https://github.com/eclipse-uprotocol/up-rust", rev = "f5248a89cf1db6232f463ee3ce7b1cb20d79cfdb" }
+up-rust = { default-features = false, git = "https://github.com/eclipse-uprotocol/up-rust", rev = "3a50104421a801d52e1d9c68979db54c013ce43d" }
```

Comment:
maybe you want to change this to
```toml
up-rust = { version = "0.1", default-features = false }
```
now that the crate is available from crates.io

Reply:
Yup, good point. Need to generally update all three of the PRs on this repo to account for that and updates to up-transport-vsomeip-rust and up-transport-zenoh-rust.

I'll do that once the PRs for up-transport-zenoh-rust and up-transport-vsomeip-rust get merged.

Reply:
Marking as resolved with intent to update in follow-up PR #26